src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -155,6 +155,16 @@
 			"# Doing so after entering data will shift things unwantedly. Using smaller value will allow higher precision,",
 			"# at the cost of more work setting up. Also, extremely small values will render the caching done useless.",
 			"# Each cell is (town_block_size * town_block_size * 128) in size, with 128 being from bedrock to clouds."),
+	TOWN_BANKRUPTSY_ENABLED(
+			"town.bankruptsy_enabled",

```

Comment:
Typo'd the word Bankruptcy. This will have to change through-out.

Reply:
Done